README.md

```diff
@@ -136,6 +136,8 @@ python3 setup.py install
 python3 setup.py develop
 ```
 
+If the commands above don't work maybe you need to install the `python-devel` and `gcc-c++` packages.
+
```

Comment:
Thsese instructions are too specific for your case. Other Linux distributions or Mac OS X users won't be able to use them. I would go for something more generic, stating the need of a compiler and python headers to run the last command successfully.